zerver/lib/actions.py

```diff
@@ -474,9 +517,8 @@ def do_deactivate_stream(stream, log=True):
                      streams=[stream_dict])
         send_event(event, active_user_ids(stream.realm))
 
-    return
-
```

Comment:
It'd be better to put any actual code changes in their own commit (which can explain the code change) rather than mixing these with adding type annotations...